doc/ref/stdlib.html

```diff
@@ -401,7 +455,7 @@ <h3 id="math">
   <div class="hgroup-inline">
     <div class="panel">
       <h4 id="std.clamp">
-        std.clamp(x, minVal, maxVal)
+        std.std.clamp(x, minVal, maxVal)
```

Comment:
Does this happen to be included by mistake?

Reply:
@ershov, did you correct this in a different PR?

Reply:
@seh no I've included it in this one

Reply:
Where is it included? When I look at the diff against the parent branch, it still shows this PR bringing in an extra ".std" on line 458.